crates/volta-core/src/tool/package/mod.rs

```diff
@@ -138,31 +138,37 @@ impl Display for Package {
 /// * the unpacked and initialized package
 pub fn uninstall(name: &str) -> Fallible<()> {
     let home = volta_home()?;
-    // if the package config file exists, use that to remove any installed bins and shims
     let package_config_file = home.default_package_config_file(name);
-    if package_config_file.exists() {
-        let package_config = PackageConfig::from_file(&package_config_file)?;
+    let package_binary_file = binaries_from_package(name)?;
 
-        for bin_name in package_config.bins {
-            remove_config_and_shim(&bin_name, name)?;
-        }
+    if package_config_file.exists() || !package_binary_file.is_empty() {
+        // if the package config file exists, use that to remove any installed bins and shims
+        if package_config_file.exists() {
+            let package_config = PackageConfig::from_file(&package_config_file)?;
 
-        fs::remove_file(&package_config_file)
-            .with_context(delete_file_error(&package_config_file))?;
-    } else {
-        // there is no package config - check for orphaned binaries
-        for bin_name in binaries_from_package(name)? {
-            remove_config_and_shim(&bin_name, name)?;
+            for bin_name in package_config.bins {
+                remove_config_and_shim(&bin_name, name)?;
+            }
+
+            fs::remove_file(&package_config_file)
+                .with_context(delete_file_error(&package_config_file))?;
+
+            info!("{} package '{}' uninstalled", success_prefix(), name);
+        } else {
+            // there is no package config - check for orphaned binaries
+            for bin_name in package_binary_file {
+                remove_config_and_shim(&bin_name, name)?;
+            }
```

Comment:
It appears we're no longer writing the "success: " message on this branch. I think it makes sense to still write it if we find _anything_ to uninstall, even if it was orphaned. Right now for this branch we don't write any sort of "done" message, just comments within the `remove_config_and_shim` method about individual binaries being uninstalled.